tez-api/src/main/java/org/apache/tez/client/TezClientUtils.java

```diff
@@ -438,6 +438,26 @@ static Credentials setupDAGCredentials(DAG dag, Credentials sessionCredentials,
     return dagCredentials;
   }
 
+  @Private
+  @VisibleForTesting
+  public static Credentials createAMCredentials(ApplicationId appId,
```

Comment:
I pulled this out because I was going to reuse but will leave it back in if I don't